Failed excel import due to missing core migrations

### Updating AB

- [X] Yes, I have [updated AB](https://github.com/LCA-ActivityBrowser/activity-browser#updating-the-ab) and still experience this issue

### What happened?

Using AB version 2.10.1, the import wizard freezes while the console shows a missing migration message.

![image](https://github.com/user-attachments/assets/35492cae-58c7-4b9c-8186-3b87a7912fb2)

I set up my project with the ecoinvent and Biosphere3 option.

![image](https://github.com/user-attachments/assets/748600fc-d168-49b9-91fe-78d0bbb79fc5)

It seems that the setup script is missing `create_core_migrations()`. The setup succeeded to load the LCIA methods, which can be used with the ecoinvent database. 

Similar to https://github.com/LCA-ActivityBrowser/activity-browser/issues/1340#issuecomment-2283621987, the project can be fixed running once:

```python
import brightway2 as bw
project = 'My project'
bw.projects.set_current(project)
bw.create_core_migrations()

```

After running once, the database can be imported with the wizard as expected.

### Relevant errors

```python
Extracted 5 worksheets in 0.15 seconds
Applying strategy: alter_database_name
Applying strategy: csv_restore_tuples
Applying strategy: csv_restore_booleans
Applying strategy: csv_numerize
Applying strategy: csv_drop_unknown
Applying strategy: csv_add_missing_exchanges_section
Applying strategy: csv_rewrite_product_key
Applying strategy: normalize_units
Applying strategy: normalize_biosphere_categories
22:00:11 | ERROR | AssertionError: Can't find migration biosphere-2-3-categories
...
File "%PathToEnv%\Lib\site-packages\bw2io\strategies\migrations.py", line 33, in migrate_exchanges
    assert migration in migrations, u"Can't find migration {}".format(migration)
```


### Operating system

Windows 10

### Conda environment

```shell
Package                  Version             Build                  Size        Kind   Source
activity-browser         2.10.1              pyhd8ed1ab_0           1.8 MiB     conda  activity-browser-2.10.1-pyhd8ed1ab_0.conda
annotated-types          0.7.0               pyhd8ed1ab_0           17.8 KiB    conda  annotated-types-0.7.0-pyhd8ed1ab_0.conda
appdirs                  1.4.4               pyh9f0ad1d_0           12.5 KiB    conda  appdirs-1.4.4-pyh9f0ad1d_0.tar.bz2
arrow                    1.3.0               pyhd8ed1ab_0           97.8 KiB    conda  arrow-1.3.0-pyhd8ed1ab_0.conda
asteval                  1.0.4               pyhd8ed1ab_0           24.9 KiB    conda  asteval-1.0.4-pyhd8ed1ab_0.conda
asttokens                2.4.1               pyhd8ed1ab_0           28.2 KiB    conda  asttokens-2.4.1-pyhd8ed1ab_0.conda
astunparse               1.6.3               pyhd8ed1ab_0           15.2 KiB    conda  astunparse-1.6.3-pyhd8ed1ab_0.tar.bz2
blinker                  1.8.2               pyhd8ed1ab_0           14.4 KiB    conda  blinker-1.8.2-pyhd8ed1ab_0.conda
brightway2               2.4.6               pyhd8ed1ab_0           9.3 KiB     conda  brightway2-2.4.6-pyhd8ed1ab_0.conda
brotli                   1.1.0               h2466b09_2             19.2 KiB    conda  brotli-1.1.0-h2466b09_2.conda
brotli-bin               1.1.0               h2466b09_2             20.3 KiB    conda  brotli-bin-1.1.0-h2466b09_2.conda
brotli-python            1.1.0               py311hda3d55a_2        314.6 KiB   conda  brotli-python-1.1.0-py311hda3d55a_2.conda
brotlicffi               1.1.0.0             py311hda3d55a_2        328.3 KiB   conda  brotlicffi-1.1.0.0-py311hda3d55a_2.conda
bw2analyzer              0.10                pyhd8ed1ab_0           28 KiB      conda  bw2analyzer-0.10-pyhd8ed1ab_0.conda
bw2calc                  1.8.2               py311h1ea47a8_0        67.8 KiB    conda  bw2calc-1.8.2-py311h1ea47a8_0.conda
bw2data                  3.6.6               pyhd8ed1ab_0           61.2 KiB    conda  bw2data-3.6.6-pyhd8ed1ab_0.conda
bw2io                    0.8.12              pyhd8ed1ab_0           17.7 MiB    conda  bw2io-0.8.12-pyhd8ed1ab_0.conda
bw2parameters            1.1.0               pyhd8ed1ab_0           19.2 KiB    conda  bw2parameters-1.1.0-pyhd8ed1ab_0.conda
bw_migrations            0.2                 pyhd8ed1ab_0           13.7 KiB    conda  bw_migrations-0.2-pyhd8ed1ab_0.conda
bzip2                    1.0.8               h2466b09_7             53.6 KiB    conda  bzip2-1.0.8-h2466b09_7.conda
ca-certificates          2024.8.30           h56e8100_0             155.1 KiB   conda  ca-certificates-2024.8.30-h56e8100_0.conda
certifi                  2024.8.30           pyhd8ed1ab_0           159.9 KiB   conda  certifi-2024.8.30-pyhd8ed1ab_0.conda
cffi                     1.17.1              py311he736701_0        290.7 KiB   conda  cffi-1.17.1-py311he736701_0.conda
charset-normalizer       3.3.2               pyhd8ed1ab_0           45.5 KiB    conda  charset-normalizer-3.3.2-pyhd8ed1ab_0.conda
click                    8.1.7               win_pyh7428d3b_0       83.1 KiB    conda  click-8.1.7-win_pyh7428d3b_0.conda
cloudpickle              3.0.0               pyhd8ed1ab_0           24.2 KiB    conda  cloudpickle-3.0.0-pyhd8ed1ab_0.conda
colorama                 0.4.6               pyhd8ed1ab_0           24.6 KiB    conda  colorama-0.4.6-pyhd8ed1ab_0.tar.bz2
comm                     0.2.2               pyhd8ed1ab_0           11.8 KiB    conda  comm-0.2.2-pyhd8ed1ab_0.conda
contourpy                1.3.0               py311h3257749_1        208.8 KiB   conda  contourpy-1.3.0-py311h3257749_1.conda
cycler                   0.12.1              pyhd8ed1ab_0           13.1 KiB    conda  cycler-0.12.1-pyhd8ed1ab_0.conda
debugpy                  1.8.5               py311hda3d55a_1        3.1 MiB     conda  debugpy-1.8.5-py311hda3d55a_1.conda
decorator                5.1.1               pyhd8ed1ab_0           11.8 KiB    conda  decorator-5.1.1-pyhd8ed1ab_0.tar.bz2
dill                     0.3.8               pyhd8ed1ab_0           86.1 KiB    conda  dill-0.3.8-pyhd8ed1ab_0.conda
docopt                   0.6.2               py_1                   14.3 KiB    conda  docopt-0.6.2-py_1.tar.bz2
ecoinvent_interface      2.4.1               pyhd8ed1ab_0           7.4 MiB     conda  ecoinvent_interface-2.4.1-pyhd8ed1ab_0.conda
eight                    1.0.1               pyhd8ed1ab_4           15.3 KiB    conda  eight-1.0.1-pyhd8ed1ab_4.conda
et_xmlfile               1.1.0               pyhd8ed1ab_0           10.4 KiB    conda  et_xmlfile-1.1.0-pyhd8ed1ab_0.conda
exceptiongroup           1.2.2               pyhd8ed1ab_0           19.9 KiB    conda  exceptiongroup-1.2.2-pyhd8ed1ab_0.conda
executing                2.1.0               pyhd8ed1ab_0           27.7 KiB    conda  executing-2.1.0-pyhd8ed1ab_0.conda
fasteners                0.17.3              pyhd8ed1ab_0           19.5 KiB    conda  fasteners-0.17.3-pyhd8ed1ab_0.tar.bz2
flask                    3.0.3               pyhd8ed1ab_0           78.9 KiB    conda  flask-3.0.3-pyhd8ed1ab_0.conda
fonttools                4.53.1              py311he736701_1        2.4 MiB     conda  fonttools-4.53.1-py311he736701_1.conda
freetype                 2.12.1              hdaf720e_2             498.3 KiB   conda  freetype-2.12.1-hdaf720e_2.conda
future                   1.0.0               pyhd8ed1ab_0           355.5 KiB   conda  future-1.0.0-pyhd8ed1ab_0.conda
glib                     2.80.3              h7025463_2             559.4 KiB   conda  glib-2.80.3-h7025463_2.conda
glib-tools               2.80.3              h4394cf3_2             92.6 KiB    conda  glib-tools-2.80.3-h4394cf3_2.conda
gst-plugins-base         1.24.7              hb0a98b8_0             2 MiB       conda  gst-plugins-base-1.24.7-hb0a98b8_0.conda
gstreamer                1.24.7              h5006eae_0             1.9 MiB     conda  gstreamer-1.24.7-h5006eae_0.conda
h2                       4.1.0               pyhd8ed1ab_0           45.7 KiB    conda  h2-4.1.0-pyhd8ed1ab_0.tar.bz2
hpack                    4.0.0               pyh9f0ad1d_0           24.7 KiB    conda  hpack-4.0.0-pyh9f0ad1d_0.tar.bz2
hyperframe               6.0.1               pyhd8ed1ab_0           14.3 KiB    conda  hyperframe-6.0.1-pyhd8ed1ab_0.tar.bz2
icu                      73.2                h63175ca_0             12.8 MiB    conda  icu-73.2-h63175ca_0.conda
idna                     3.10                pyhd8ed1ab_0           48.7 KiB    conda  idna-3.10-pyhd8ed1ab_0.conda
importlib-metadata       8.5.0               pyha770c72_0           28 KiB      conda  importlib-metadata-8.5.0-pyha770c72_0.conda
importlib_metadata       8.5.0               hd8ed1ab_0             9.2 KiB     conda  importlib_metadata-8.5.0-hd8ed1ab_0.conda
inflate64                1.0.0               py311he736701_2        38.8 KiB    conda  inflate64-1.0.0-py311he736701_2.conda
intel-openmp             2023.2.0            h57928b3_50497         2.4 MiB     conda  intel-openmp-2023.2.0-h57928b3_50497.conda
ipykernel                6.29.5              pyh4bbf305_0           117 KiB     conda  ipykernel-6.29.5-pyh4bbf305_0.conda
ipython                  8.27.0              pyh7428d3b_0           586.1 KiB   conda  ipython-8.27.0-pyh7428d3b_0.conda
itsdangerous             2.2.0               pyhd8ed1ab_0           18.9 KiB    conda  itsdangerous-2.2.0-pyhd8ed1ab_0.conda
jedi                     0.19.1              pyhd8ed1ab_0           821.6 KiB   conda  jedi-0.19.1-pyhd8ed1ab_0.conda
jinja2                   3.1.4               pyhd8ed1ab_0           109 KiB     conda  jinja2-3.1.4-pyhd8ed1ab_0.conda
jupyter_client           8.6.2               pyhd8ed1ab_0           103.8 KiB   conda  jupyter_client-8.6.2-pyhd8ed1ab_0.conda
jupyter_core             5.7.2               py311h1ea47a8_0        109.1 KiB   conda  jupyter_core-5.7.2-py311h1ea47a8_0.conda
kiwisolver               1.4.7               py311h3257749_0        54.6 KiB    conda  kiwisolver-1.4.7-py311h3257749_0.conda
krb5                     1.21.3              hdf4eb48_0             695.3 KiB   conda  krb5-1.21.3-hdf4eb48_0.conda
lcms2                    2.16                h67d730c_0             495.7 KiB   conda  lcms2-2.16-h67d730c_0.conda
lerc                     4.0.0               h63175ca_0             189.8 KiB   conda  lerc-4.0.0-h63175ca_0.tar.bz2
libblas                  3.9.0               20_win64_mkl           4.8 MiB     conda  libblas-3.9.0-20_win64_mkl.conda
libbrotlicommon          1.1.0               h2466b09_2             68.9 KiB    conda  libbrotlicommon-1.1.0-h2466b09_2.conda
libbrotlidec             1.1.0               h2466b09_2             31.9 KiB    conda  libbrotlidec-1.1.0-h2466b09_2.conda
libbrotlienc             1.1.0               h2466b09_2             240.2 KiB   conda  libbrotlienc-1.1.0-h2466b09_2.conda
libcblas                 3.9.0               20_win64_mkl           4.8 MiB     conda  libcblas-3.9.0-20_win64_mkl.conda
libclang13               18.1.8              default_ha5278ca_4     24.1 MiB    conda  libclang13-18.1.8-default_ha5278ca_4.conda
libdeflate               1.21                h2466b09_0             152.1 KiB   conda  libdeflate-1.21-h2466b09_0.conda
libexpat                 2.6.3               he0c23c2_0             135.7 KiB   conda  libexpat-2.6.3-he0c23c2_0.conda
libffi                   3.4.2               h8ffe710_5             41.1 KiB    conda  libffi-3.4.2-h8ffe710_5.tar.bz2
libglib                  2.80.3              h7025463_2             3.6 MiB     conda  libglib-2.80.3-h7025463_2.conda
libhwloc                 2.11.1              default_h8125262_1000  2.3 MiB     conda  libhwloc-2.11.1-default_h8125262_1000.conda
libiconv                 1.17                hcfcfb64_2             621.2 KiB   conda  libiconv-1.17-hcfcfb64_2.conda
libintl                  0.22.5              h5728263_3             93.3 KiB    conda  libintl-0.22.5-h5728263_3.conda
libintl-devel            0.22.5              h5728263_3             39.8 KiB    conda  libintl-devel-0.22.5-h5728263_3.conda
libjpeg-turbo            3.0.0               hcfcfb64_1             803.7 KiB   conda  libjpeg-turbo-3.0.0-hcfcfb64_1.conda
liblapack                3.9.0               20_win64_mkl           4.8 MiB     conda  liblapack-3.9.0-20_win64_mkl.conda
libogg                   1.3.5               h2466b09_0             34.6 KiB    conda  libogg-1.3.5-h2466b09_0.conda
libpng                   1.6.44              h3ca93ac_0             340.8 KiB   conda  libpng-1.6.44-h3ca93ac_0.conda
libsodium                1.0.20              hc70643c_0             197.6 KiB   conda  libsodium-1.0.20-hc70643c_0.conda
libsqlite                3.46.1              h2466b09_0             856.1 KiB   conda  libsqlite-3.46.1-h2466b09_0.conda
libtiff                  4.6.0               hb151862_4             766.3 KiB   conda  libtiff-4.6.0-hb151862_4.conda
libvorbis                1.3.7               h0e60522_0             267.3 KiB   conda  libvorbis-1.3.7-h0e60522_0.tar.bz2
libwebp                  1.4.0               h2466b09_0             69.4 KiB    conda  libwebp-1.4.0-h2466b09_0.conda
libwebp-base             1.4.0               hcfcfb64_0             267.9 KiB   conda  libwebp-base-1.4.0-hcfcfb64_0.conda
libxcb                   1.16                h013a479_1             966.3 KiB   conda  libxcb-1.16-h013a479_1.conda
libxml2                  2.12.7              h0f24e4e_4             1.6 MiB     conda  libxml2-2.12.7-h0f24e4e_4.conda
libxslt                  1.1.39              h3df6e99_0             408.7 KiB   conda  libxslt-1.1.39-h3df6e99_0.conda
libzlib                  1.3.1               h2466b09_1             54.9 KiB    conda  libzlib-1.3.1-h2466b09_1.conda
lxml                     4.9.3               py311h750ae06_3        1007.3 KiB  conda  lxml-4.9.3-py311h750ae06_3.conda
lxmlh                    1.3.3               pyhd8ed1ab_0           12.1 KiB    conda  lxmlh-1.3.3-pyhd8ed1ab_0.conda
m2w64-gcc-libgfortran    5.3.0               6                      342.5 KiB   conda  m2w64-gcc-libgfortran-5.3.0-6.tar.bz2
m2w64-gcc-libs           5.3.0               7                      519.9 KiB   conda  m2w64-gcc-libs-5.3.0-7.tar.bz2
m2w64-gcc-libs-core      5.3.0               7                      214.1 KiB   conda  m2w64-gcc-libs-core-5.3.0-7.tar.bz2
m2w64-gmp                6.1.0               2                      726.1 KiB   conda  m2w64-gmp-6.1.0-2.tar.bz2
m2w64-libwinpthread-git  5.0.0.4634.697f757  2                      31.2 KiB    conda  m2w64-libwinpthread-git-5.0.0.4634.697f757-2.tar.bz2
markupsafe               2.1.5               py311he736701_1        29.3 KiB    conda  markupsafe-2.1.5-py311he736701_1.conda
matplotlib-base          3.9.2               py311h8f1b1e4_1        7.5 MiB     conda  matplotlib-base-3.9.2-py311h8f1b1e4_1.conda
matplotlib-inline        0.1.7               pyhd8ed1ab_0           14.3 KiB    conda  matplotlib-inline-0.1.7-pyhd8ed1ab_0.conda
mkl                      2023.2.0            h6a75c08_50497         138 MiB     conda  mkl-2023.2.0-h6a75c08_50497.conda
mrio_common_metadata     0.2.1               pyhd8ed1ab_0           16.5 KiB    conda  mrio_common_metadata-0.2.1-pyhd8ed1ab_0.tar.bz2
msys2-conda-epoch        20160418            1                      3.2 KiB     conda  msys2-conda-epoch-20160418-1.tar.bz2
multiprocess             0.70.16             py311he736701_1        368 KiB     conda  multiprocess-0.70.16-py311he736701_1.conda
multivolumefile          0.2.3               pyhd8ed1ab_0           20.6 KiB    conda  multivolumefile-0.2.3-pyhd8ed1ab_0.tar.bz2
munkres                  1.1.4               pyh9f0ad1d_0           12.2 KiB    conda  munkres-1.1.4-pyh9f0ad1d_0.tar.bz2
nest-asyncio             1.6.0               pyhd8ed1ab_0           11.4 KiB    conda  nest-asyncio-1.6.0-pyhd8ed1ab_0.conda
networkx                 3.3                 pyhd8ed1ab_1           1.1 MiB     conda  networkx-3.3-pyhd8ed1ab_1.conda
numpy                    1.26.4              py311h0b4df5a_0        6.8 MiB     conda  numpy-1.26.4-py311h0b4df5a_0.conda
openjpeg                 2.5.2               h3d672ee_0             232.4 KiB   conda  openjpeg-2.5.2-h3d672ee_0.conda
openpyxl                 3.1.5               py311ha68e1ae_1        632.9 KiB   conda  openpyxl-3.1.5-py311ha68e1ae_1.conda
openssl                  3.3.2               h2466b09_0             8 MiB       conda  openssl-3.3.2-h2466b09_0.conda
packaging                24.1                pyhd8ed1ab_0           49.1 KiB    conda  packaging-24.1-pyhd8ed1ab_0.conda
pandas                   2.2.2               py311hcf9f919_1        13.9 MiB    conda  pandas-2.2.2-py311hcf9f919_1.conda
parso                    0.8.4               pyhd8ed1ab_0           73.4 KiB    conda  parso-0.8.4-pyhd8ed1ab_0.conda
pathos                   0.3.2               pyhd8ed1ab_1           51.6 KiB    conda  pathos-0.3.2-pyhd8ed1ab_1.conda
patsy                    0.5.6               pyhd8ed1ab_0           182.8 KiB   conda  patsy-0.5.6-pyhd8ed1ab_0.conda
pcre2                    10.44               h3d7b363_2             801.6 KiB   conda  pcre2-10.44-h3d7b363_2.conda
peewee                   3.17.6              py311h1eea93c_1        466.9 KiB   conda  peewee-3.17.6-py311h1eea93c_1.conda
pickleshare              0.7.5               py_1003                9.1 KiB     conda  pickleshare-0.7.5-py_1003.tar.bz2
pillow                   10.4.0              py311h5592be9_1        40.2 MiB    conda  pillow-10.4.0-py311h5592be9_1.conda
pint                     0.21                pyhd8ed1ab_0           218.7 KiB   conda  pint-0.21-pyhd8ed1ab_0.conda
pip                      24.2                pyh8b19718_1           1.2 MiB     conda  pip-24.2-pyh8b19718_1.conda
platformdirs             4.3.3               pyhd8ed1ab_0           20.1 KiB    conda  platformdirs-4.3.3-pyhd8ed1ab_0.conda
pox                      0.3.4               pyhd8ed1ab_0           26 KiB      conda  pox-0.3.4-pyhd8ed1ab_0.conda
ppft                     1.7.6.8             pyhd8ed1ab_0           33.3 KiB    conda  ppft-1.7.6.8-pyhd8ed1ab_0.conda
prompt-toolkit           3.0.47              pyha770c72_0           264.4 KiB   conda  prompt-toolkit-3.0.47-pyha770c72_0.conda
psutil                   6.0.0               py311he736701_1        513.8 KiB   conda  psutil-6.0.0-py311he736701_1.conda
pthread-stubs            0.4                 hcd874cb_1001          6.3 KiB     conda  pthread-stubs-0.4-hcd874cb_1001.tar.bz2
pthreads-win32           2.9.1               hfa6e2cd_3             140.9 KiB   conda  pthreads-win32-2.9.1-hfa6e2cd_3.tar.bz2
pure_eval                0.2.3               pyhd8ed1ab_0           16.2 KiB    conda  pure_eval-0.2.3-pyhd8ed1ab_0.conda
py7zr                    0.22.0              pyhd8ed1ab_0           58.9 KiB    conda  py7zr-0.22.0-pyhd8ed1ab_0.conda
pybcj                    1.0.2               py311he736701_1        35 KiB      conda  pybcj-1.0.2-py311he736701_1.conda
pycasreg                 0.1.0               pyhd8ed1ab_0           10.4 KiB    conda  pycasreg-0.1.0-pyhd8ed1ab_0.conda
pycparser                2.22                pyhd8ed1ab_0           102.6 KiB   conda  pycparser-2.22-pyhd8ed1ab_0.conda
pycryptodomex            3.20.0              py311he736701_0        1.6 MiB     conda  pycryptodomex-3.20.0-py311he736701_0.conda
pydantic                 2.9.1               pyhd8ed1ab_0           293.6 KiB   conda  pydantic-2.9.1-pyhd8ed1ab_0.conda
pydantic-core            2.23.3              py311h533ab2d_0        1.5 MiB     conda  pydantic-core-2.23.3-py311h533ab2d_0.conda
pydantic-settings        2.5.2               pyhd8ed1ab_0           28.5 KiB    conda  pydantic-settings-2.5.2-pyhd8ed1ab_0.conda
pyecospold               3.5.3               pyhd8ed1ab_0           68.2 KiB    conda  pyecospold-3.5.3-pyhd8ed1ab_0.conda
pygments                 2.18.0              pyhd8ed1ab_0           858.7 KiB   conda  pygments-2.18.0-pyhd8ed1ab_0.conda
pypardiso                0.4.6               pyhd8ed1ab_1           14.8 KiB    conda  pypardiso-0.4.6-pyhd8ed1ab_1.conda
pyparsing                3.1.4               pyhd8ed1ab_0           88 KiB      conda  pyparsing-3.1.4-pyhd8ed1ab_0.conda
pyperclip                1.9.0               pyh7428d3b_0           16.5 KiB    conda  pyperclip-1.9.0-pyh7428d3b_0.conda
pyppmd                   1.1.0               py311hda3d55a_1        61.5 KiB    conda  pyppmd-1.1.0-py311hda3d55a_1.conda
pyprind                  2.11.2              py311h1ea47a8_1006     18.7 KiB    conda  pyprind-2.11.2-py311h1ea47a8_1006.conda
pyside2                  5.15.8              py311h3d699ce_4        8.1 MiB     conda  pyside2-5.15.8-py311h3d699ce_4.conda
pysocks                  1.7.1               pyh0701188_6           18.9 KiB    conda  pysocks-1.7.1-pyh0701188_6.tar.bz2
python                   3.11.10             hce54a09_0_cpython     17.4 MiB    conda  python-3.11.10-hce54a09_0_cpython.conda
python-dateutil          2.9.0               pyhd8ed1ab_0           217.5 KiB   conda  python-dateutil-2.9.0-pyhd8ed1ab_0.conda
python-dotenv            1.0.1               pyhd8ed1ab_0           23.7 KiB    conda  python-dotenv-1.0.1-pyhd8ed1ab_0.conda
python-tzdata            2024.1              pyhd8ed1ab_0           140.6 KiB   conda  python-tzdata-2024.1-pyhd8ed1ab_0.conda
python_abi               3.11                5_cp311                6.5 KiB     conda  python_abi-3.11-5_cp311.conda
pytz                     2024.2              pyhd8ed1ab_0           182.6 KiB   conda  pytz-2024.2-pyhd8ed1ab_0.conda
pywin32                  306                 py311h12c1d0e_2        5.8 MiB     conda  pywin32-306-py311h12c1d0e_2.conda
pyxlsb                   1.0.10              pyhd8ed1ab_0           27.6 KiB    conda  pyxlsb-1.0.10-pyhd8ed1ab_0.tar.bz2
pyzmq                    26.2.0              py311h484c95c_2        361.2 KiB   conda  pyzmq-26.2.0-py311h484c95c_2.conda
pyzstd                   0.16.1              py311h53056dc_0        100.3 KiB   conda  pyzstd-0.16.1-py311h53056dc_0.conda
qhull                    2020.2              hc790b64_5             1.3 MiB     conda  qhull-2020.2-hc790b64_5.conda
qt-main                  5.15.8              h06adc49_23            57.5 MiB    conda  qt-main-5.15.8-h06adc49_23.conda
qt-webengine             5.15.8              h4bf5c4e_4             62.1 MiB    conda  qt-webengine-5.15.8-h4bf5c4e_4.tar.bz2
requests                 2.32.3              pyhd8ed1ab_0           57.4 KiB    conda  requests-2.32.3-pyhd8ed1ab_0.conda
salib                    1.5.1               pyhd8ed1ab_0           648.5 KiB   conda  salib-1.5.1-pyhd8ed1ab_0.conda
scipy                    1.14.1              py311hd4686c6_0        15.5 MiB    conda  scipy-1.14.1-py311hd4686c6_0.conda
seaborn                  0.13.2              hd8ed1ab_2             6.8 KiB     conda  seaborn-0.13.2-hd8ed1ab_2.conda
seaborn-base             0.13.2              pyhd8ed1ab_2           229.1 KiB   conda  seaborn-base-0.13.2-pyhd8ed1ab_2.conda
setuptools               73.0.1              pyhd8ed1ab_0           1.4 MiB     conda  setuptools-73.0.1-pyhd8ed1ab_0.conda
setuptools-scm           8.1.0               pyhd8ed1ab_0           36.9 KiB    conda  setuptools-scm-8.1.0-pyhd8ed1ab_0.conda
six                      1.16.0              pyh6c4a22f_0           13.9 KiB    conda  six-1.16.0-pyh6c4a22f_0.tar.bz2
spyder-kernels           3.0.0               win_pyh7428d3b_0       214.2 KiB   conda  spyder-kernels-3.0.0-win_pyh7428d3b_0.conda
stack_data               0.6.2               pyhd8ed1ab_0           25.6 KiB    conda  stack_data-0.6.2-pyhd8ed1ab_0.conda
stats_arrays             0.7                 pyhd8ed1ab_0           23.7 KiB    conda  stats_arrays-0.7-pyhd8ed1ab_0.conda
statsmodels              0.14.2              py311h0a17f05_0        11.3 MiB    conda  statsmodels-0.14.2-py311h0a17f05_0.conda
tabulate                 0.9.0               pyhd8ed1ab_1           35.1 KiB    conda  tabulate-0.9.0-pyhd8ed1ab_1.tar.bz2
tbb                      2021.13.0           hc790b64_0             147.9 KiB   conda  tbb-2021.13.0-hc790b64_0.conda
texttable                1.7.0               pyhd8ed1ab_0           19.4 KiB    conda  texttable-1.7.0-pyhd8ed1ab_0.conda
tk                       8.6.13              h5226925_1             3.3 MiB     conda  tk-8.6.13-h5226925_1.conda
tomli                    2.0.1               pyhd8ed1ab_0           15.6 KiB    conda  tomli-2.0.1-pyhd8ed1ab_0.tar.bz2
tornado                  6.4.1               py311he736701_1        840.6 KiB   conda  tornado-6.4.1-py311he736701_1.conda
tqdm                     4.66.5              pyhd8ed1ab_0           87.4 KiB    conda  tqdm-4.66.5-pyhd8ed1ab_0.conda
traitlets                5.14.3              pyhd8ed1ab_0           107.6 KiB   conda  traitlets-5.14.3-pyhd8ed1ab_0.conda
types-python-dateutil    2.9.0.20240906      pyhd8ed1ab_0           21.3 KiB    conda  types-python-dateutil-2.9.0.20240906-pyhd8ed1ab_0.conda
typing-extensions        4.12.2              hd8ed1ab_0             9.9 KiB     conda  typing-extensions-4.12.2-hd8ed1ab_0.conda
typing_extensions        4.12.2              pyha770c72_0           39 KiB      conda  typing_extensions-4.12.2-pyha770c72_0.conda
tzdata                   2024a               h8827d51_1             121.3 KiB   conda  tzdata-2024a-h8827d51_1.conda
ucrt                     10.0.22621.0        h57928b3_0             1.2 MiB     conda  ucrt-10.0.22621.0-h57928b3_0.tar.bz2
unicodecsv               0.14.1              pyhd8ed1ab_2           16.4 KiB    conda  unicodecsv-0.14.1-pyhd8ed1ab_2.conda
unidecode                1.3.8               pyhd8ed1ab_0           169.7 KiB   conda  unidecode-1.3.8-pyhd8ed1ab_0.conda
urllib3                  2.2.2               pyhd8ed1ab_1           92.8 KiB    conda  urllib3-2.2.2-pyhd8ed1ab_1.conda
vc                       14.3                h8a93ad2_21            16.8 KiB    conda  vc-14.3-h8a93ad2_21.conda
vc14_runtime             14.40.33810         ha82c5b3_21            734.1 KiB   conda  vc14_runtime-14.40.33810-ha82c5b3_21.conda
voluptuous               0.15.2              pyhd8ed1ab_1           32.5 KiB    conda  voluptuous-0.15.2-pyhd8ed1ab_1.conda
vs2015_runtime           14.40.33810         h3bf8584_21            16.8 KiB    conda  vs2015_runtime-14.40.33810-h3bf8584_21.conda
wcwidth                  0.2.13              pyhd8ed1ab_0           31.9 KiB    conda  wcwidth-0.2.13-pyhd8ed1ab_0.conda
werkzeug                 3.0.4               pyhd8ed1ab_0           236.6 KiB   conda  werkzeug-3.0.4-pyhd8ed1ab_0.conda
wheel                    0.44.0              pyhd8ed1ab_0           57.2 KiB    conda  wheel-0.44.0-pyhd8ed1ab_0.conda
whoosh                   2.7.4               py311h1ea47a8_9        815.7 KiB   conda  whoosh-2.7.4-py311h1ea47a8_9.conda
win_inet_pton            1.1.0               pyhd8ed1ab_6           8 KiB       conda  win_inet_pton-1.1.0-pyhd8ed1ab_6.tar.bz2
wrapt                    1.16.0              py311he736701_1        61 KiB      conda  wrapt-1.16.0-py311he736701_1.conda
xlrd                     2.0.1               pyhd8ed1ab_3           91.9 KiB    conda  xlrd-2.0.1-pyhd8ed1ab_3.tar.bz2
xlsxwriter               3.2.0               pyhd8ed1ab_0           120 KiB     conda  xlsxwriter-3.2.0-pyhd8ed1ab_0.conda
xorg-libxau              1.0.11              hcd874cb_0             50.1 KiB    conda  xorg-libxau-1.0.11-hcd874cb_0.conda
xorg-libxdmcp            1.1.3               hcd874cb_0             66.3 KiB    conda  xorg-libxdmcp-1.1.3-hcd874cb_0.tar.bz2
xz                       5.2.6               h8d14728_0             212.7 KiB   conda  xz-5.2.6-h8d14728_0.tar.bz2
zeromq                   4.3.5               he1f189c_5             2.6 MiB     conda  zeromq-4.3.5-he1f189c_5.conda
zipfile-deflate64        0.2.0               py311ha68e1ae_4        26.7 KiB    conda  zipfile-deflate64-0.2.0-py311ha68e1ae_4.conda
zipp                     3.20.2              pyhd8ed1ab_0           20.9 KiB    conda  zipp-3.20.2-pyhd8ed1ab_0.conda
zstandard                0.23.0              py311h53056dc_1        313.8 KiB   conda  zstandard-0.23.0-py311h53056dc_1.conda
zstd                     1.5.6               h0ea2cb4_0             341 KiB     conda  zstd-1.5.6-h0ea2cb4_0.conda
```

--- a/activity_browser/actions/__init__.py
+++ b/activity_browser/actions/__init__.py
@@ -58,3 +58,4 @@ from .default_install import DefaultInstall
 from .biosphere_update import BiosphereUpdate
 from .plugin_wizard_open import PluginWizardOpen
 from .settings_wizard_open import SettingsWizardOpen
+from .migrations_install import MigrationsInstall

--- a/activity_browser/actions/migrations_install.py
+++ b/activity_browser/actions/migrations_install.py
@@ -1,0 +1,40 @@
+from PySide2 import QtWidgets
+
+from activity_browser import application
+from activity_browser.actions.base import ABAction, exception_dialogs
+from activity_browser.ui import icons, threading
+from activity_browser.mod.bw2io.migrations import ab_create_core_migrations
+
+
+class MigrationsInstall(ABAction):
+    """
+    ABAction to install the default migrations from bw2io
+    """
+
+    icon = icons.qicons.import_db
+    text = "Install default migrations"
+
+    @staticmethod
+    @exception_dialogs
+    def run():
+        def update_dialog_slot(progress: int, label: str):
+            dialog.setValue(progress)
+            dialog.setLabelText(label)
+
+
+        dialog = QtWidgets.QProgressDialog(application.main_window)
+        dialog.setWindowTitle("Installing migrations")
+        dialog.setMaximum(100)
+        dialog.setCancelButton(None)
+
+        thread = MigrationsInstallThread(application)
+
+        thread.status.connect(update_dialog_slot)
+
+        thread.start()
+        dialog.exec_()
+
+
+class MigrationsInstallThread(threading.ABThread):
+    def run_safely(self):
+        ab_create_core_migrations()

--- a/activity_browser/mod/bw2io/ecoinvent.py
+++ b/activity_browser/mod/bw2io/ecoinvent.py
@@ -15,8 +15,8 @@ def ab_import_ecoinvent_release(version, system_model):
     from bw2io import migrations
     from .migrations import ab_create_core_migrations
 
-    # if not len(migrations):
-    #     ab_create_core_migrations()
+    if not len(migrations):
+        ab_create_core_migrations()
 
     # downloading a release through a AB version of ecoinvent_interface that implements a progress_slot
     release = ABEcoinventRelease(ei.Settings())

--- a/activity_browser/ui/menu_bar.py
+++ b/activity_browser/ui/menu_bar.py
@@ -57,6 +57,8 @@ class MenuBar(QtWidgets.QMenuBar):
         self.file_menu.addAction(self.export_db_action)
         self.file_menu.addAction(self.update_biosphere_action)
         self.file_menu.addSeparator()
+        self.file_menu.addMenu(MigrationsMenu(self))
+        self.file_menu.addSeparator()
         self.file_menu.addAction(self.manage_settings_action)
 
     def setup_view_menu(self) -> None:
@@ -121,3 +123,16 @@ For license information please see the copyright on <a href="https://github.com/
         exists = True if bd.config.biosphere in bd.databases else False
         self.update_biosphere_action.setEnabled(exists)
         self.import_db_action.setEnabled(exists)
+
+
+class MigrationsMenu(QtWidgets.QMenu):
+    def __init__(self, parent=None) -> None:
+        super().__init__(parent)
+
+        self.setTitle("Migrations")
+        self.install_migrations_action = actions.MigrationsInstall.get_QAction()
+
+        self.addAction(self.install_migrations_action)
+
+
+

--- a/tests/actions/test_various_actions.py
+++ b/tests/actions/test_various_actions.py
@@ -1,6 +1,7 @@
 import os
 
 import bw2data as bd
+import bw2io as bi
 import pytest
 from PySide2 import QtWidgets
 
@@ -66,3 +67,17 @@ def test_settings_wizard_open(ab_app):
     assert application.main_window.findChild(SettingsWizard).isVisible()
 
     application.main_window.findChild(SettingsWizard).destroy()
+
+
+def test_migrations_install(ab_app, qtbot):
+    from activity_browser.actions.migrations_install import MigrationsInstallThread
+
+    project_name = "migrations_project"
+    bd.projects.set_current(project_name)
+
+    assert not len(bi.migrations)
+
+    actions.MigrationsInstall.run()
+
+    assert len(bi.migrations)
+

